### DISCUSSION TITLE
Support Multi-cursor mode across multiple cells

### DISCUSSION THREAD

**Discussion by ydmykr** [microsoft/vscode-jupyter#11186] [New Ideas]
Recently, I inquired about the operation of multiple cells, but I did not find the operation of editing multiple cells at the same time. Even `Alt + Shift + leftclick` supported in vscode does not work in multiple cells.

Now suppose I have multiple cells, and I want to use the magic command `%%capture` on the top of all cells.

**Comment by rebornix** [comment]
We currently didn't support this yet, it's being tracked in https://github.com/microsoft/vscode/issues/141673
